src/components/SideBar/SideBar.styles.ts

```diff
@@ -0,0 +1,41 @@
+import {
```

Comment:
You should prefix all styled components with component name, eg. Root => SidebarRoot, StyledDrawer => SidebarSyledDrawer